<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_key</class>
 <widget class="QDialog" name="Dialog_key">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="user" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>188</width>
     <height>45</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="user_label">
      <property name="text">
       <string>user:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="user_text"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="password" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>219</width>
     <height>45</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="password_label">
      <property name="text">
       <string>password:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="password_text"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="buttons" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>176</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="cancel_btn">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ok_btn">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
